app/models/task.py

```diff
@@ -3,4 +3,7 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default=None)
```

Comment:
add `goal_id` adding a `db.ForeignKey("goal.goal_id")` to connect it to the Goal class